src/test/features/dashboard/claims-state-machine/initial-transitions.ts

```diff
@@ -0,0 +1,54 @@
+import { expect } from 'chai'
+import { Claim } from 'claims/models/claim'
+import { sampleClaimIssueObj } from 'test/http-mocks/claim-store'
+
+import { initialTransitions } from 'dashboard/claims-state-machine/initial-transitions'
+import { MomentFactory } from 'shared/momentFactory'
+import { ResponseType } from 'claims/models/response/responseType'
+
+describe('State Machine for the dashboard status before response', () => {
+  describe('given the claim with no response', () => {
+    it('should extract the correct state for the claim issued', () => {
+      const claim: Claim = new Claim().deserialize(sampleClaimIssueObj)
+      let claimState = initialTransitions(claim)
+      claimState.findState(claimState)
+      expect(claimState.state).to.equal('no-response')
+    })
+  })
+
+  describe('given the claim with more time requested', () => {
+    it('should extract the correct state for the claim issued', () => {
+      const claim: Claim = new Claim().deserialize({ ...sampleClaimIssueObj,moreTimeRequested: true })
```

Comment:
Missing space after comma here. and other places in the same file. please reformat this as well.

Reply:
I will format the code with CMC code style

Reply:
Its done ... committed changes